src/res/translation/wininstaller/pl.nsi

```diff
@@ -0,0 +1,37 @@
+; Polish translation
+
+LangString DESKTOP_SET_SHORTCUT ${LANG_POLISH} \
```

Comment:
I hope the LANG_POLISH is correct here.

Reply:
I found from where it goes: `res/translation/wininstaller/installerlng.nsi`
So there is incantation for every supported installer lang:
```
!insertmacro MUI_LANGUAGE "Dutch"
!include "${ROOT_PATH}\src\res\translation\wininstaller\nl.nsi"
```

In NSIS this `!insertmacro MUI_LANGUAGE` thing adds suitable variable `${LANG_SOMETHING}` for any language.

so I may add the same for the Polish one.